LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -10,9 +10,8 @@ import LoopKit
 import HealthKit
 
 public class TherapySettingsViewModel: ObservableObject {
-    
-    private var initialTherapySettings: TherapySettings
-    public var therapySettings: TherapySettings
+    public var initialTherapySettings: TherapySettings
```

Comment:
So we can reset the initialTherapySettings within the therapy settings flow

Reply:
Why can't you use the `reset()` function?

Reply:
You're 100% correct, changed